README.md

```diff
@@ -245,6 +245,10 @@ Includes the content of a file or a group of files.
 - <a name="include_encoding" href="#include_encoding">#</a>
   **encoding** (_utf-8_): Specify the encoding of the included file.
   If not defined `utf-8` will be used.
+  **recursive** (_true_): When this option is disabled, it will not try process
+  included files recursively. This can be useful when you want to include complex
+  HTML contents such as [Plotly](https://plotly.com/graphing-libraries/) figures.
+  Possible values are `true` and `false`.
```

Comment:
Please, don't put the Plotly example here, is not relevant. And the wording should be "it will not process", remove the "try".